cli.py

```diff
@@ -68,8 +71,9 @@ def add_role(username: str, role: str):
 
 @cli.command()
 @click.argument("title")
-def create_test_project(title):
-    """Create a test proofing project with 100 pages."""
+@click.argument("pdf_path")
+def create_project(title, pdf_path):
```

Comment:
Should we document this and other CLI commands elsewhere?

Reply:
Yes, will do so in a follow-up PR (tracking in #204).